geo-types/src/coordinate.rs

```diff
@@ -30,7 +30,18 @@ pub struct Coordinate<T: CoordNum> {
     pub y: T,
 }
 
+impl<T: CoordNum> Coordinate<T> {
+    /// Create a new instance of a coordinate.
+    /// **ATTENTION** Use [`coord!`] macro or one of the [`from`] methods instead.
+    #[inline]
+    #[doc(hidden)]
+    pub fn new__(x: T, y: T) -> Self {
```

Comment:
The fn new__ is the same as the prev. impl of the macro.  Why is it causing perf. regression?

Reply:
I have no idea, but i did see the numbers change. Regardless, I will move this change to another PR to research separately from this PR.